agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/init/FirstEntryPoint.java

```diff
@@ -103,8 +103,7 @@ public void init() {
       StatusFile.startupLogger = startupLogger;
       ConfigurationBuilder.logConfigurationWarnMessages();
 
-      AppIdSupplier appIdSupplier = AiComponentInstaller.beforeAgent();
-      StartAppIdRetrieval.setAppIdSupplier(appIdSupplier);
+      LegacyInstrumentation.install();
```

Comment:
can we rename install to transform? be consistent with other Instrumentation type of classes?

Reply:
this method doesn't do the transform, it registers the transformers, maybe `registerTransformers()`?